prodigy/util/order.py

```diff
@@ -0,0 +1,34 @@
+from typing import Iterator, List
+
+
+def default_monomial_iterator(n: int) -> Iterator[List[int]]:
```

Comment:
Does this iterator ensures that every possible tuple is enumerated in finite time? I.e. doe it enumerated them "diagonally"?

Reply:
Its the same iterator used before and on first glance seems to enumerate all tuples, I can take a look if you want